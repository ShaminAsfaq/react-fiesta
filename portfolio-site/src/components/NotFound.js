import React from 'react';
import { Link } from 'react-router-dom';

const _404 = () => {
    return (
        <div>
            You are in the 404! - <Link to="/">Go Home!</Link>
        </div>
    );
}

export {
    _404 as default
}

