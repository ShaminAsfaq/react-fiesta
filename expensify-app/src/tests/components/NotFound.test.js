import React from 'react';
import NotFound from '../../components/NotFound';
import { shallow } from 'enzyme';

test('Testing NotFoundComponent', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();

    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />)
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
})


 