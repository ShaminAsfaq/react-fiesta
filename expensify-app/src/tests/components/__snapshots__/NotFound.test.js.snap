// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing NotFoundComponent 1`] = `
<div>
  You are in the 404! - 
  <Link
    replace={false}
    to="/"
  >
    Go Home!
  </Link>
</div>
`;
